content/md/retailer-onboarder/first-steps/retailer-synchronization.md

```diff
@@ -16,21 +16,35 @@ You will find here all the necessary information related to synchronization:
 
-During the supplier creation you will be able to link one or n families to a supplier.
+When you create a new supplier, you need to link families and catalog locales to a supplier.
 What does it mean and what is the impact?
 
-If you link the supplier "Abibas" to the "Shoes" and "Accessories" families, it means that he will supply you with the products belonging to these families, so you expect him to fill in product data for these 2 families.
+If you link the supplier "Abibas" to the "Shoes" and "Accessories" families, it means that they will supply you with the products belonging to these families in all the locales you selected, so you expect the supplier to fill in product data for these 2 families.
 Moreover, this link will avoid displaying attributes the Supplier does not work on in the Supplier Onboarder User Interface.
 
-As a consequence, the family, the attribute groups, attributes and attribute options will be synchronized with the relevant Supplier Onboarder meaning that he will be able to filter on the product grid using these attributes and to create products belonging to this family. In addition to that, in the PIM Product Form, only the suppliers linked to the product family will be displayed in the supplier dropdown.
+Consequently, the following information will be synchronized with the relevant Supplier Onboarder for all selected locales:
+* families,
+* attribute groups,
+* attributes (if localizable)
+* attribute options (if localizable).
+
+<!--
+As a consequence, the family, the attribute groups, attributes and attribute options will be synchronized with the relevant Supplier Onboarder in all the selected locales if attributes are localizable.
+-->
+
+It means that suppliers are able to filter on the product grid using these attributes and to create products belonging to this family. In addition to that, in the PIM Product Form, only the suppliers linked to the product family will be displayed in the supplier dropdown.
 
 ::: info
 Catalog structure updates are also synchronized with the relevant Supplier Onboarder. For example, if you add a new required attribute for a family for the Onboarder channel, it will be automatically synchronized on the relevant Supplier Onboarder.
 :::
 
 # Products synchronization
 ## Products
-As soon as you link a product to a supplier and you set a supplier reference for a product, the attributes mentioned above will be synchronized with the relevant Supplier Onboarder. Therefore, the product will be displayed in the relevant supplier Onboarder product grid.
+As soon as you link a product to a supplier and you set a supplier reference for a product, the attributes mentioned above will be synchronized with the relevant Supplier Onboarder. Therefore, the product will be displayed in the supplier Onboarder product grid.
+
+::: info
+Suppliers are able to enrich data of *localizable* attributes when they are required on the channel Onboarder and the catalog locales are linked to them.
```

Comment:
this sentence is a bit complicated, and not really easy to understand.

Reply:
```suggestion
To be enriched by a supplier, a **localizable** attribute has to be set as required on the Onboarder channel.  The locales need to be activated at the channel level and linked for each supplier. 
```

What do you think?

Reply:
```suggestion
To be enriched by a supplier, a **localizable** attribute has to be set as required on the Onboarder channel.  The locales need to be activated at the channel level and linked to each supplier. 
```